### PR TITLE
Planning: update jerk bounds

### PR DESCRIPTION
**Pull request by HongyiSun**

### CODE REVIEW

File: modules/planning/tasks/optimizers/piecewise_jerk_path/piecewise_jerk_path_optimizer.cc [hunk 1]
```diff
@@ -170,15 +170,15 @@ bool PiecewiseJerkPathOptimizer::OptimizePath(
...
 
-  /**
-  // Experimental code to be tested
-  // TODO(all): find the params in vehicle config
-  double axis_distance = 2.5;
-  double max_steering_rate = 1.0 / 6.0 * M_PI;
+  // Estimate jerk boundary from vehicle_params
+  const auto& veh_param =
+      common::VehicleConfigHelper::GetConfig().vehicle_param();
+  double axis_distance = veh_param.wheel_base();
```
Comment (Capri2014) [inline modules/planning/tasks/optimizers/piecewise_jerk_path/piecewise_jerk_path_optimizer.cc]: const

File: modules/planning/tasks/optimizers/piecewise_jerk_path/piecewise_jerk_path_optimizer.cc [hunk 2]
```diff
@@ -170,15 +170,15 @@ bool PiecewiseJerkPathOptimizer::OptimizePath(
...
+  double max_steering_rate = veh_param.max_steer_angle_rate() /
```
Comment (Capri2014) [inline modules/planning/tasks/optimizers/piecewise_jerk_path/piecewise_jerk_path_optimizer.cc]: Did you mean heading angle change rate here?

Comment (Capri2014) [reply modules/planning/tasks/optimizers/piecewise_jerk_path/piecewise_jerk_path_optimizer.cc]: Or wheel angle change rate?

Comment (HongyiSun) [reply modules/planning/tasks/optimizers/piecewise_jerk_path/piecewise_jerk_path_optimizer.cc]: fixed, thanks!

**Pull request merged**